models/models_methods.go

```diff
@@ -450,6 +470,74 @@ func (config *BitriseDataModel) Validate() ([]string, error) {
 	}
 	// ---
 
+	// pipelines
```

Comment:
Please extract the newly introduced validation parts (pipelines, stages) to separate functions to improve readability.